include/DummyEffect.h

```diff
@@ -61,8 +61,7 @@ class DummyEffectControls : public EffectControls
 	}
 
 	~DummyEffectControls() override
-	{
-	}
+	= default;
```

Comment:
Nope?

Reply:
No idea what you're talking about. :smirk: